src/components/LaunchFilter/constants/filerConstants.ts

```diff
@@ -0,0 +1,16 @@
+export enum LaunchFilters {
+  PREVIOUS = 'PREVIOUS',
+  CREWED = 'CREWED',
+  CA_LOCATION = 'CA_LOCATION',
+  FL_LOCATION = 'FL_LOCATION',
+}
+
+export enum FilterCaptions {
+  PREVIOUS = 'Previous launches',
+  CREWED = 'Crewed launches',
+  CA_LOCATION = 'Location: California',
+  FL_LOCATION = 'Location: Florida',
+}
+
+export const CA_LOCATION = '11';
+export const FL_LOCATION = '12,27';
```

Comment:
Which location is this?

Reply:
11 - Vandenberg SFB, CA, USA
12 - Cape Canaveral, FL, USA
27 - Kennedy Space Center, FL, USA